website_video_preview/static/src/js/website_video_preview.editor.js

```diff
@@ -92,8 +96,8 @@ odoo.define('website_video_preview.editor', function(require) {
          * @private
          * @returns {String}
          */
-        _getYoutubePreviewImage: function(id) {
-            return 'https://img.youtube.com/vi/' + id + '/0.jpg';
+        _getYoutubePreviewImage: function (id) {
+            return "https://img.youtube.com/vi/" + id + "/0.jpg";
```

Comment:
This is OK as it is, but just in case you prefer it:

```suggestion
            return _.str.sprintf("https://img.youtube.com/vi/%s/0.jpg", id);
```